src/util/createElement.js

```diff
@@ -1,6 +1,6 @@
 /**
  * Create a DOM element with various attributes
- *
+ * 
```

Comment:
Could help to add husky to the project and run prettier automatically on every commit

Reply:
I like that idea @bozdoz